### DISCUSSION TITLE
Allowing Chats with Traders before taking offer [Closed]

### DISCUSSION THREAD

**Discussion by Conza88** [bisq-network/bisq#5177] [Ideas]
There's a variety of important communication improvements to trader chat that have already been proposed[1], this however proposes to give the **_OPTION_** for potential traders to chat with the MAKER of an offer _before taking it._ The maker would have the choice as to whether they want to offer the ability to be engaged in pre-offer chat, or not. 

There's been several occasions where I've seen an offer outside my fiat range, or in another payment method I do not have access to. Perhaps a % slightly too high premium for my liking etc. Having been able to contact the MAKER he may have been willing to amend his offer with new terms that are acceptable.

![image](https://user-images.githubusercontent.com/51952482/107344752-ea57b080-6b16-11eb-8a3f-c02e9cb97866.png)

For this to be helpful, the other proposed trader chat (communication improvements) changes should take priority. 

**Possible Negatives**
- Efforts to low ball traders offers, tying up and wasting maker's time with 'offers' or useless chat
- Spammy comments
- Arguments against: this is a consequence to a lack of liquidity / ability to get trades from other markets etc. Something #5147 would certainly go towards fixing.

**Possible Positives**
- Trades happen that otherwise wouldn't.

Putting it out there to discuss. Any thoughts welcomed.

----------------------------------------------------------------
[1] 
- Improve "Portfolio" icon to aid communication between traders #5074 
- Add ability for mediator's & arbitrator's to see "Trader Chat" (upon escalation) #5044 
- Add Notification icon for Mediation and Arbitration chat #5097

**Comment by huey735** [comment]
As I mention on [Sign-Verify, Encrypt-Decrypt and onion addresses](https://github.com/bisq-network/bisq/discussions/5174) it may be worth it to use other established end-to-end-encrypted-apps for chatting. And once in that app the traders willing to chat could simply join a channel with their onion address.

**Comment by pazza83** [comment]
I am not keen on people being able to ask questions prior to trade taking place. Especially if the only reasons if around price.

If the offer a trader wants to is not there I think this is a good incentive for them to create their own offer.

It would be frustrating to be communicating with someone only for the offer to be come and taken by someone else.

I also think the ability of only being able to communicate with a trader where there is an active trade between peers is good from a privacy view point. Last thing I want is people messaging me that have not accepted trades asking questions about my bank details, name, what language I speak, where I am located etc

**Reply by Conza88** [reply]
Simple answer is **_CHOOSE_** to not enable it then. 

I don't think anyone would be asking about bank details or anything like that. 

Having said that: _"If the offer a trader wants to is not there I think this is a good incentive for them to create their own offer."_

That I can accept. I'd close this.. but can't lol #discussions

**Comment by pazza83** [comment]
![closed](https://user-images.githubusercontent.com/66367547/107866697-53b72680-6e6b-11eb-8cd7-61e756dfb148.jpg)
